packages/terra-framework-docs/src/terra-dev-site/doc/notification-dialog/example/NotificationDialogWithLongText.jsx

```diff
@@ -48,7 +49,7 @@ const NotificationDialogWithLongText = () => {
         <NotificationDialog
           variant="hazard-medium"
           dialogTitle="Use a Title That Relates Directly to the Actions - Long Titles Should Be Avoided When Possible"
-          startMessage={message}
+          startMessage={<ContentContainer setFocusOnContainer>{message}</ContentContainer>}
```

Comment:
startMessage prop is string type. You cannot have a node here.

Reply:
startMessage is also accepting type node, since content is node type replaced it with content.